src/Jobs/Recruit.Vacancies.Jobs/QueueTriggers/GenerateBlockedEmployersQueueTrigger.cs

```diff
@@ -5,31 +5,28 @@
 using Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.BlockedEmployers;
 using Esfa.Recruit.Vacancies.Client.Infrastructure.StorageQueue;
 using Esfa.Recruit.Vacancies.Jobs.Configuration;
+using Esfa.Recruit.Vacancies.Jobs.UpdateBlockedEmployers;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Extensions.Logging;
 
-namespace Esfa.Recruit.Vacancies.Jobs.NonLevyAccountBlocker
+namespace Esfa.Recruit.Vacancies.Jobs.QueueTriggers
 {
-    public class NonLevyAccountBlockerJob
+    public class GenerateBlockedEmployersQueueTrigger
```

Comment:
Where we have used the disabledJobs configuration you will need to check on each environment and update that field to have the new name.

Reply:
👍